code/modules/jobs/job_types/roguetown/church/templar.dm

```diff
@@ -293,3 +288,29 @@
 			H.put_in_hands(new /obj/item/rogueweapon/katar/abyssor(H), TRUE)
 			H.adjust_skillrank(/datum/skill/combat/unarmed, 1, TRUE)
 			ADD_TRAIT(H, TRAIT_CIVILIZEDBARBARIAN, TRAIT_GENERIC)
+
+//Helmets. This BLOWS with how it checks, but I'm lazy and it works.
```

Comment:
Ideally at some point we'll be able to get visored helms for the other helmets to make it so check isn't really needed-
But for now this will suffice. I do reccomend changing this comment to mention overhauling once we get sprites for others or something similar, however.

Reply:
Can do. I'll adjust the comment when I get the chance in a bit here.